website/docs/r/spring_cloud_application_insights_application_performance_monitoring.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+subcategory: "Spring Cloud"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_application_insights_application_performance_monitoring"
+description: |-
+  Manages a Spring Cloud Application Insights Application Performance Monitoring.
+---
+
+# azurerm_spring_cloud_application_insights_application_performance_monitoring
+
+-> **NOTE:** This resource is applicable only for Spring Cloud Service with enterprise tier.
+
+Manages a Spring Cloud Application Insights Application Performance Monitoring.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+resource "azurerm_application_insights" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  application_type    = "web"
+}
+
+resource "azurerm_spring_cloud_service" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "E0"
+}
+
+resource "azurerm_spring_cloud_application_insights_application_performance_monitoring" "example" {
+  name                         = "example"
+  spring_cloud_service_id      = azurerm_spring_cloud_service.example.id
+  connection_string            = azurerm_application_insights.example.instrumentation_key
+  globally_enabled             = true
+  role_name                    = "test-role"
+  role_instance                = "test-instance"
+  sampling_percentage          = 50
+  sampling_requests_per_second = 10
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Spring Cloud Application Insights Application Performance Monitoring. Changing this forces a new resource to be created.
```

Comment:
change to "The name which should be used for this Spring Cloud Application Performance Monitoring resource for Application Insights"